src/__tests__/middleware.test.ts

```diff
@@ -0,0 +1,73 @@
+import { ApolloServer } from '@apollo/server';
+import type { APIGatewayProxyEventV2 } from 'aws-lambda';
+import { handlers, startServerAndCreateLambdaHandler } from '..';
+
+const event: APIGatewayProxyEventV2 = {
```

Comment:
Can this be shared with the other integration test? If we end up having to update one for correctness it'd be nice not to have duplication.

Reply:
I'm not sure if sharing is possible as the tests in this case are dependent on a static event, while the others are dependent on the incoming request. Maybe I'm misunderstanding?